### PR TITLE
[EarlyReturn] Add ChangeAndIfContinueToMultiContinueRector

### LINKED ISSUE #5230: [EarlyReturn] Split && if to 2 continues 2
```diff
   foreach ($this->dimFetchAssignToMethodCalls as $dimFetchAssignToMethodCall) {
-            if ($this->isObjectType($arrayDimFetch->var, $dimFetchAssignToMethodCall->getListClass()) && $this->isObjectType($assign->expr, $dimFetchAssignToMethodCall->getItemClass())) {
+            if (! $this->isObjectType($arrayDimFetch->var, $dimFetchAssignToMethodCall->getListClass())) {
+                      continue;
+              }
+				if (! $this->isObjectType($assign->expr, $dimFetchAssignToMethodCall->getItemClass()))) {
+ 			{
+               continue;
+			}
                return $dimFetchAssignToMethodCall;
-            }
        }
```

### PR DESCRIPTION
**Pull request by samsonasik**
Fixes #5230

### CODE REVIEW

**Comment by samsonasik** [comment]
implemented, moved re-use functionality for create if negation to `IfManipulator` service.

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

**Comment by samsonasik** [comment]
rebased.

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: src/NodeManipulator/IfManipulator.php [hunk 1]
```diff
@@ -336,6 +325,33 @@ public function isIfCondUsingAssignNotIdenticalVariable(If_ $if, Node $node): bo
...
+        if ($expr instanceof Identical) {
+            return new NotIdentical($expr->left, $expr->right);
+        }
+
+        if ($expr instanceof NotIdentical) {
+            return new Identical($expr->left, $expr->right);
+        }
+
+        return new BooleanNot($expr);
+    }
```
Comment (TomasVotruba) [inline src/NodeManipulator/IfManipulator.php:353]: Might be useful `Rector\EarlyReturn\NodeTransformer\ConditionInverter`

Comment (samsonasik) [reply src/NodeManipulator/IfManipulator.php]: implemented.

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

**Comment by samsonasik** [comment]
rebased.

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**